### ISSUE TITLE
bug: block quotes lose underlying styles

### ISSUE THREAD

**Issue by calebdw** [mfontanini/presenterm#341]
Hello!

It seems that block quotes lose any styles which might be inside the quote. For instance, given the following markdown:

```markdown
> Here is a **blockquote**, which can be used to *emphasize* certain parts of your `presentation`.
```

here is how most software will render this:

> Here is a **blockquote**, which can be used to *emphasize* certain parts of your `presentation`.

and here is how presenterm renders:

![image](https://github.com/user-attachments/assets/6c303971-14b8-4331-93cc-441a48b053b7)

Thanks!

**Comment by mfontanini** [comment]
This isn't really a bug but a conscious decision I made while implementing this because I didn't have the tools to do this properly. Now after some recent changes, preserving the style should be simpler. I started implementing this but it's a bit trickier than I expected. I'll have a look at it this week a bit more.

**Comment by mfontanini** [comment]
@calebdw I created https://github.com/mfontanini/presenterm/pull/350 to address this. Can you give it a shot and let me know if anything looks odd? The only thing that I'm aware of but unfortunately can't fix is multiple lines get truncated to 1:

```markdown
> first
>
>
>
>
> last
```

Will be rendered as:

> first
> 
> last

But it seems this is how the markdown parsing library I'm using delivers the AST to me so I don't think I can get around this. This issue exists in the current implementation anyway so it's not a new thing.

**Comment by calebdw** [comment]
@mfontanini, thanks!

Sorry it took so long to respond, it's been busy at work.

The behavior is better, unfortunately it looks like the inline code is being ignored (no styles to `presentation`):

![image](https://github.com/user-attachments/assets/fe471892-57b7-487d-a61a-db051db7d5c3)

**Comment by mfontanini** [comment]
Dammit, I checked bold, underline, and strike through but didn't check inline code. Will fix tomorrow. Thanks for checking!

**Comment by mfontanini** [comment]
Fixed in #351
